.github/workflows/smoke.yml

```diff
@@ -265,13 +265,22 @@ jobs:
         LOCAL_GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
         set -o pipefail
-        ./dependabot test \
-          -f=smoke.yaml \
-          -o=result.yaml \
-          --cache=cache \
-          --timeout=20m \
-          --updater-image=ghcr.io/dependabot/dependabot-updater-${{ matrix.suite.ecosystem }}:latest \
-          2>&1 | tee -a log.txt
+        if [[ "${{matrix.suite.name}}" == "docker" ]]; then
+          ./dependabot test \
+            -f=smoke.yaml \
+            -o=result.yaml \
+            --timeout=20m \
+            --updater-image=ghcr.io/dependabot/dependabot-updater-${{ matrix.suite.ecosystem }}:latest \
+            2>&1 | tee -a log.txt
```

Comment:
We can get rid of this exception, I can merge the PR and update the cache after 👍

Reply:
I just added it to confirm the test succeeds if we disable the cache. Removed the commit.